src/ui/sidebar/index.html

```diff
@@ -126,14 +126,14 @@ <h6>Springs</h6>
       <img src="../trussfab-globals/assets/icons/spring.png" width="54px" alt="spring">
       <div>spring</div>
     </button>
-    <button type="button" class="tool" id="spring_animation_tool">
+    <button type="button" class="tool" id="place_user_tool">
       <img src="../trussfab-globals/assets/icons/sitting_child.png" width="54px" alt="spring">
       <div>place user</div>
     </button>
-<!--    <button type="button" class="tool" id="spring_simulation_tool">-->
-<!--      <img src="../trussfab-globals/assets/icons/spring.png" width="54px" alt="spring">-->
-<!--      <div>simulate</div>-->
-<!--    </button>-->
```

Comment:
You can comment it out by just wrapping it in one ```<!-- {...} -->``` that's what we have done for the other buttons

Reply:
Yes but I removed it :-)